src/components/ui/explore/ExploreFilter.tsx

```
@@ -15,6 +22,62 @@ export default function ExploreFilter() {
   const regionList = ['남구'];
   const stationList = ['못골역', '국제금융센터·부산은행역', '문현역', '지게골역', '대연역', '경성대·부경대역'];
 
+  const getLocation = (name: string, type: 'school' | 'station'): Location => {
```

ESLint

[error] 'type' is defined but never used.

(@typescript-eslint/no-unused-vars)